linux/LinuxProcessList.c

```diff
@@ -1727,10 +1728,28 @@ static int scanCPUFreqencyFromSysCPUFreq(LinuxProcessList* this) {
    int numCPUsWithFrequency = 0;
    unsigned long totalFrequency = 0;
 
+   /*
+    * On some AMD CPUs read()ing scaling_cur_freq is quite slow (> 1ms).
```

Comment:
Just to be accurate.
As you could see in https://github.com/htop-dev/htop/issues/471#issuecomment-759378878 this is not AMD specific issue. At least some of the Intel Xeon CPUs also share such behavior.